src/Analyzers/CSharp/Analyzers/UseConditionalExpression/CSharpUseConditionalExpressionForReturnDiagnosticAnalyzer.cs

```diff
@@ -21,5 +22,25 @@ public CSharpUseConditionalExpressionForReturnDiagnosticAnalyzer()
 
         protected override ISyntaxFacts GetSyntaxFacts()
             => CSharpSyntaxFacts.Instance;
+
+        protected override bool AreStatementsSupported(IOperation trueStatement, IOperation falseStatement)
+        {
+            // Return statements wrapped in an unsafe, checked or unchecked block are not supported
```

Comment:
so, this is just a statement of what the code is doing.  Consider instead saying why.